src/polypartition.cpp

```diff
@@ -109,17 +97,17 @@ int TPPLPoly::GetOrientation() const {
     area += points[i1].x * points[i2].y - points[i1].y * points[i2].x;
   }
   if (area > 0) {
-    return TPPL_CCW;
+    return TPPL_ORIENTATION_CCW;
   }
   if (area < 0) {
-    return TPPL_CW;
+    return TPPL_ORIENTATION_CW;
   }
-  return 0;
+  return TPPL_ORIENTATION_NONE;
 }
 
-void TPPLPoly::SetOrientation(int orientation) {
-  int polyorientation = GetOrientation();
-  if (polyorientation && (polyorientation != orientation)) {
+void TPPLPoly::SetOrientation(TPPLOrientation orientation) {
+  TPPLOrientation polyorientation = GetOrientation();
+  if (__glibc_likely(polyorientation != TPPL_ORIENTATION_NONE) && (polyorientation != orientation)) {
```

Comment:
Will `__glibc_likely` work on all platforms though? I don't think it's part of the standard.

Reply:
Good catch, it won't. I removed it. I'm surprised that the standard doesn't include anything like this.